### ISSUE TITLE
using scroll bar causes freezes

### ISSUE THREAD

**Issue by tfriedel** [akiyosi/goneovim#551]
I enabled the scroll bar and opened a large file (100s of pages) and after using the scroll bar to scroll a large portion the app freezes. 
Also using the scroll bar felt unnatural. I.e. the scrolling of the text didn't keep up with the scroll bar and it felt like I couldn't move as fast as I wanted.

**Comment by akiyosi** [comment]
@tfriedel 
If you start goneovim without Neovim personal settings as follows, will the issue be the same?

```
\path\to\goneovim -u NONE
```

**Comment by tfriedel** [comment]
no, freezes were not observed when running without personal settings.
The personal settings I tested it with were with an almost vanilla lazyvim installation. 

Btw, I have meanwhile switched to using nvim-scrollview which provides a working scroll bar.
